### PR TITLE
add nterview Prep in Navbar

### PR DESCRIPTION
**Pull request by harshad00**
#59

### CODE REVIEW

File: src/constant/global.ts [hunk 1]
```diff
@@ -31,6 +31,11 @@ const products: ProductDataProps = {
     slug: '/shiksha',
     description: 'Learn Tech with Free Bite-sized Courses',
   },
+  interviewPrep: {
+    label: 'InterviewPrep',
+    slug: '/interviewPrep',
```
Comment (imsks) [inline src/constant/global.ts]: slug should be "/interview-prep". Follow existing code stye

File: src/interfaces/global.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ export type ProductLabelType =
   | 'Roadmaps'
   | 'The Boring Projects'
   | 'Shiksha'
+  | 'InterviewPrep'
```
Comment (imsks) [inline src/interfaces/global.ts]: Interview Prep not InterviewPrep

**Review by imsks — CHANGES_REQUESTED** [review]

**Comment by harshad00** [comment]
maybe now we will not have issues and have any problems tall me.

**Comment by imsks** [comment]
We've fixed and merged this already. Please pick another task.

**Pull request closed without merging**